Docs/mc_definition.tex

```diff
@@ -110,13 +110,7 @@ \section{Configuration Management}
 
 Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) "hookup" connections and (2) "physical" connections, discussed below.
 
-To capture antenna status, an {\em a priori} antenna status table has been defined to facilitate a work flow to incorporate new antennas into the array and capture and known recalcitrant antennas.  The table provides an enumerated status for each antenna station.  The status is one of:
-\begin{itemize}\setlength\itemsep{-.3em}
-	\item not\_connected
-	\item needs\_checking
-	\item known\_bad
-	\item passed\_checks
-\end{itemize}
+To capture antenna status, an {\em a priori} antenna status table has been defined to facilitate a work flow to incorporate new antennas into the array and capture and known recalcitrant antennas.  The table provides an enumerated status for each antenna station.
 
 
 \subsection{Operation Workflow}
```

Comment:
The text in this subsection is out of date.

Reply:
Updated.